recipes/splinecalib/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License

```

Comment:
Could you please create an issue or PR upstream to request the LICENSE to be included in next release?

Reply:
numeristical/splinecalib#1